recipe/meta.yaml

```diff
@@ -47,7 +47,9 @@ requirements:
     # See libevent variants' definition in `recipe/conda_build_config.yaml`
     - libevent
     - libprotobuf
-    - libmongocxx
+    # TODO: Fix builds for missing symbols
+    # See: https://github.com/man-group/ArcticDB/pull/1050
+    - libmongocxx <3.9
```

Comment:
We should check that mamba installs this build or ArcticDB and not an older one when requiring ArcticDB and other package that have common dependencies. If not, we may want to mark previous builds as broken.

Reply:
Actually we are safe since:
 - [`libmongocxx`'s runtime exports set a constraints on minor versions](https://github.com/conda-forge/libmongocxx-feedstock/blob/main/recipe/meta.yaml#L15)
 - no build of ArcticDB against `libmongocxx>=3.9` has been published (for instance `arcticdb-4.1.0-py39h32bbf47_2.conda`, one of the last builds, depends on `libmongocxx >=3.7.2,<3.8.0a0`, see [this list](https://prefix.dev/channels/conda-forge/packages/arcticdb))
 
Still I think we _must_ prevent this sort of issue by pinning minor version of all host and build dependencies. What do you think?

Reply:
> Actually we are safe since: [ ... ]

Ok, thanks for this precision.

> Still I think we must prevent this kind of issue by pinning minor versions of all host and build dependencies.

Actually I think pinning should be done on the major version only (whose change ensures backward incompatibility); ABI compatibility through minor versions should be achieved with run exports. Pinning mior version should be done as a last resort.

Reply:
OK. https://github.com/conda-forge/arcticdb-feedstock/issues/116 has been opened to continue discussions.